### DISCUSSION TITLE
Hide "Quick Switcher" Move in Menu

### DISCUSSION THREAD

**Discussion by scpedicini** [tadashi-aikawa/obsidian-another-quick-switcher#261] [General]
This is a pretty minor ask, but is there any way we could get an option in the settings to remove/hide Quick Switcher's "Move File"? This feature already exists as part of the official Obsidian system.

![CleanShot 2024-11-08 at 15 31 11@2x](https://github.com/user-attachments/assets/ecc72000-f2c1-49cd-823f-139cc7181d8a)

**Comment by tadashi-aikawa** [comment]
@scpedicini 
There is no such method in Another Quick Switcher. You can achieve this with a plugin like [Obsidian Better Command Palette](https://github.com/AlexBieg/obsidian-better-command-palette), which has a feature to hide certain commands.

However, as you mentioned, if there is an official command available, I believe it's safe to delete this command 🤔 

Background on the initial addition: https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/issues/8#issuecomment-954377161

**Reply by tadashi-aikawa** [reply]
I’ll keep this command as it is. 
https://github.com/tadashi-aikawa/obsidian-another-quick-switcher/issues/260#issuecomment-2466067005

**Reply by scpedicini** [reply]
@tadashi-aikawa Sounds good. Does seem like there's a very slight difference. My only fear was more around liability since moving files is a more risky operation as it is inherently "write" based rather than the majority of the usage of Quick-switcher which is "read" based.

**Reply by tadashi-aikawa** [reply]
Certainly, I've felt some discomfort all along, but this expression feels the closest 🤔
